### ISSUE TITLE
Screenshot issue in report on mobile landscape and Chrome emulated devices

### ISSUE THREAD

**Issue by vijkris99** [galenframework/galen#432]
Galen has problems with screenshots in the report if the device orientation is changed from its default value using Appium (For example, running a test on a Samsung Galaxy S4 in Landscape mode - see snapshot below). 

![mobile landscape issue](https://cloud.githubusercontent.com/assets/18624257/16876671/cf11eff8-4a72-11e6-8656-62a7241404e6.png)

Galen has similar screenshot issues running on devices emulated using Chrome browser.

![chrome emulation issue](https://cloud.githubusercontent.com/assets/18624257/16876670/cf0f1b20-4a72-11e6-9eb0-f76fc9899ff6.png)

In both cases, the tool itself seems to be working fine, so this is probably just an issue with the report itself. These issues occur irrespective of the specs used or the website tested, so it should be possible to reproduce this fairly easily. Do let me know if you would like more details.

**Comment by ishubin** [comment]
Thanks for notifying! We used to have similar issues a long time ago. They were related to device pixel ratio and as far as I remember they were fixed. Perhaps this is a regression

**Comment by vijkris99** [comment]
Yes, maybe it is a regression defect... It would be great if this could get fixed, considering that Galen's reports are one of the high points of the framework :)

**Comment by ishubin** [comment]
@vijkris99 Actually I just realized that this might not be an issue of Galen. It seems that Appium gives you incorrect image. For instance on the first photo I clearly see that the phone is in a landscape orientation but Appium provides an image in a weird-state  (content is in portrait but image is cropped like it is in landscape).
Also on the second image I see that there is a grey area on the image which doesn't make a lot of sense. In Galen we don't have such functionality and we don't fill the rest of the image with grey color.

**Comment by ankitdadhich** [comment]
@vijkris99 How to launch mobile site while testing with galen? In my current scenario it launches desktop site always.
